scripts/build-probe-binaries

```diff
@@ -373,9 +373,11 @@ function rhel_build {
 
 		HASH_ORIG=$HASH
 
+		export KERNELDIR=$BASEDIR/$KERNEL_RELEASE/usr/src/kernels/$KERNEL_RELEASE
+		cd "${KERNELDIR}" && make gcc-plugins || true
```

Comment:
Why do we need to `|| true`? Is it to handle kernels that don't have plugins? We want to know early as possible if there are errors.

Related, is this actually RHEL-specific, or are they simply the first kernel using the plugins? Based on my reading of https://www.kernel.org/doc/Documentation/gcc-plugins.txt, could we instead do this in `build_probe` but only if `$(src)/scripts/gcc-plugins` is populated?